gateway/README.md

```diff
@@ -301,6 +302,35 @@ For example, with the config below, `documentId` will be replaced with the prope
         }
     } 
 
+##### Removing Span Tag Data
+
+If a tag contains sensitive data that needs to be removed, `RemoveSpanTags` can be used to define tags that should be removed from specific services/operations. 
```

Comment:
Simplify:

> `RemoveSpanTags` can be used to remove certain tags from the received trace spans. This can be used if you expect certain tags to contain sensitive information that you want to remove from your trace spans. The tags to remove can be specified by service name and operation name; both support using `*` for wildcard matching.